admin/src/utils/ita.js

```diff
@@ -40,6 +41,8 @@ function getCategoryDisplayName(category) {
       return "Images";
     case "theme":
       return "Theme";
+    case "themes":
+      return "Themes";
```

Comment:
I might have called this `themes_json` because it seems to do the same thing as `theme` just in a different format.

Reply:
`"theme"` is how the cloud admins used to setup custom colors in the old UI. That's going to be deprecated when I merge https://github.com/mozilla/hubs/pull/4030 

I did not remove `"theme"` for two reasons: 
1) I think it might still be used for determining in-world menu/sprite colors. (I need to check this).
2) Users might not have a copy of their colors elsewhere. If we remove the "theme" menu, they might lose this data. By keeping it in, they have an easier migration path for writing the json blob they need to migrate their old colors to the new UI.

As you said -- `themes` is a list of available themes to select from as you said. Selecting one of these themes puts your preference into local storage but does not affect the `theme` config.

Reply:
Update: I updated the in-world menu/sprite colors so that they use the new `themes` config values.